.github/workflows/ci.yml

```diff
@@ -90,7 +90,10 @@ jobs:
       - name: 'Bazel: fetch'
         run: bazel fetch //tensorboard/...
       - name: 'Bazel: build'
-        run: bazel build //tensorboard/...
+        run: |
```

Comment:
Is there a way to comment in a yml file? Can you comment about why you're supressing the error message?

Similarly, is there a place we can leave a comment near where the real fix should go? Then we can comment that this kludge should be removed?